libpod/container.go

```diff
@@ -424,6 +424,9 @@ type ContainerConfig struct {
 	// to 0, 1, 2) that will be passed to the executed process. The total FDs
 	// passed will be 3 + PreserveFDs.
 	PreserveFDs uint `json:"preserveFds,omitempty"`
+
+	//Tz is the timezone inside the container
+	TZ string `json:"timezone"`
```

Comment:
omitempty

Reply:
Please rename to "Timezone" as part of this struct - brevity is good for CLI options, not good for code clarity.